src/DETHRACE/common/mainloop.c

```diff
@@ -664,6 +664,10 @@ tRace_result MainGameLoop() {
     if (gAction_replay_mode) {
         ToggleReplay();
     }
+    // From splatpack x-mas demo
```

Comment:
True, but I think calling this function unconditionally is ok here.
Otherwise, you'll start the race as an arrow if you happened to have ended the previous race as an arrow.